### PR TITLE
3653: fix member.getBalance to doc

### PR DESCRIPTION
**Pull request by pavlushev**
What I did:
- fix member.getBalance to doc

### CODE REVIEW

File: logicrunner/builtin/proxy/deposit/deposit.go [hunk 1]
```diff
@@ -31,15 +31,16 @@ type DaemonConfirm struct {
...
+	PulseDepositUnHold      int64                  `json:"holdReleaseDate"`
 	MigrationDaemonConfirms []DaemonConfirm        `json:"confirmerReferences"`
 	Amount                  string                 `json:"amount"`
 	TxHash                  string                 `json:"ethTxHash"`
 	VestingType             foundation.VestingType `json:"vestingType"`
-	MaturePulse             insolar.PulseNumber    `json:"maturePulse"`
-	Lockup                  int64                  `json:"lockupInPulses"`
-	Vesting                 int64                  `json:"vestingInPulses"`
-	VestingStep             int64                  `json:"vestingStepInPulses"`
+	MaturePulse             int64                  `json:"matureDate"`
```
Comment (ivanshibitov) [inline logicrunner/builtin/proxy/deposit/deposit.go]: "MaturePulse" and "matureDate" are inconsistent. Let's name them equally

File: logicrunner/builtin/contract/deposit/deposit.go [hunk 2]
```diff
@@ -67,17 +67,27 @@ type DaemonConfirm struct {
 
 func (d Deposit) toOut() DepositOut {
 	var daemonConfirms = make([]DaemonConfirm, 0, len(d.MigrationDaemonConfirms))
+	var pulseDepositUnHold, matureTime int64
 	for k, v := range d.MigrationDaemonConfirms {
 		daemonConfirms = append(daemonConfirms, DaemonConfirm{Reference: k, Amount: v})
 	}
+	t, err := d.PulseDepositUnHold.AsApproximateTime()
+	if err == nil {
+		pulseDepositUnHold = t.UnixNano()
```
Comment (ivanshibitov) [inline logicrunner/builtin/contract/deposit/deposit.go]: Why Nano?
What timestamp format has to be here?

Comment (ivanshibitov) [reply logicrunner/builtin/contract/deposit/deposit.go]: In spec it's not nano

File: logicrunner/builtin/contract/deposit/deposit.go [hunk 3]
```diff
@@ -67,17 +67,27 @@ type DaemonConfirm struct {
...
 	var daemonConfirms = make([]DaemonConfirm, 0, len(d.MigrationDaemonConfirms))
+	var pulseDepositUnHold, matureDate int64
 	for k, v := range d.MigrationDaemonConfirms {
 		daemonConfirms = append(daemonConfirms, DaemonConfirm{Reference: k, Amount: v})
 	}
+	t, err := d.PulseDepositUnHold.AsApproximateTime()
+	if err == nil {
+		pulseDepositUnHold = t.Unix()
+	}
+	mature, err := d.MaturePulse.AsApproximateTime()
```
Comment (ivanshibitov) [inline logicrunner/builtin/contract/deposit/deposit.go]: MaturePulse - он нигде не устанавливается.
В выводе получится 0 всегда

Comment (pavlushev) [reply logicrunner/builtin/contract/deposit/deposit.go]: It is established on those deposits that are created on the genesis for various funds.

**Pull request merged**